### DISCUSSION TITLE
Does NWS API use static IPs?

### DISCUSSION THREAD

**Discussion by hnabbasi** [weather-gov/api#637] [Q&A] [answered]
I know it's a bit odd question to ask, but I was asked if the APIs can be pinged directly to underlying IPs? if the IPs are static or dynamic. The client has some restrictions on what web address it can call out of the network, and we would like to know if NWS APIs use static IPs.

Thank you.

**Comment by jah1007** [comment, marked as answer]
The API is fronted by a Content Delivery Network (CDN) and thus does not have a static IP.

**Reply by hnabbasi** [reply]
Thank you.
